### PR TITLE
Removed unused variable from CSCSegAlgoRU

### PR DESCRIPTION
**Pull request by Dr15Jones**
This fixes a clang warning.

### CODE REVIEW

**Comment by Dr15Jones** [comment]
please test

**Comment by perrotta** [comment]
+1
- Technical
- Jenkins tests pass

File: RecoLocalMuon/CSCSegment/src/CSCSegAlgoRU.cc [hunk 1]
```diff
@@ -560,7 +560,6 @@ float CSCSegAlgoRU::fit_r_phi(const AlgoState& aState, const SVector6& points, i
 
 void CSCSegAlgoRU::baseline(AlgoState& aState, int n_seg_min) const {
   int nhits = aState.proto_segment.size();
-  ChamberHitContainer::const_iterator iRH_worst;
```
Comment (slava77) [inline RecoLocalMuon/CSCSegment/src/CSCSegAlgoRU.cc]: this is already done in #25684 

to avoid a need to rebase there may I propose to drop this PR.

Comment (Dr15Jones) [reply RecoLocalMuon/CSCSegment/src/CSCSegAlgoRU.cc]: I'll close this then.

**Comment by Dr15Jones** [comment]
change is already in another pull request.

**Comment by slava77** [comment]
@Dr15Jones 
I'm not sure how long it will take to converge on #25684
At this moment, I think that there will be no significant harm from a need to rebase there.
So, feel free to reopen/remake this.

**Comment by Dr15Jones** [comment]
@slava77 there is no hurry on my end. I just wanted this to eventually get fixed which will presumably happen in the other pull request.

**Pull request closed without merging**